intro.rst

```diff
@@ -0,0 +1,16 @@
+Introduction
+============
+
+Behaviour Driven Development
+----------------------------
+
+Once you're up and running with Behat, you can learn more about behaviour
+driven development via the following links. Though both tutorials are specific
+to Cucumber, Behat shares a lot with Cucumber and the philosophies are one
+and the same.
```

Comment:
Well, we should not rely only on the side menu IMO. The doc content itself should allow navigation.

Here I'm quite sure that people will continue by moving to the external links, which are very visible when reading the content